lib/liquid/tags/for.rb

```diff
@@ -145,14 +144,14 @@ def collection_segment(context)
       segment = Utils.slice_collection(collection, from, to)
       segment.reverse! if @reversed
```

Comment:
`Range` doesn't have `reverse!` (or `reverse`), could you add a test for that?

Reply:
`range.max.downto(range.min)` should also work there, no?

Reply:
Hm, actually that would make `empty?` checking difficult. I'll throw together a custom enumerator.